src/activities/GianaBhateja/GianaBhateja.qml

```diff
@@ -0,0 +1,575 @@
+        anchors.horizontalCenter: parent.horizontalCenter
+    }*/
+    Text {
+        id: textLabel
+        anchors.centerIn: parent
+        text: "Drag    and    Drop    Here    in    a    row......."
+    }
+    pageComponent: Image {
+        id: background
+        source: "qrc:/gcompris/src/activities/GianaBhateja/resource/img2.svg"
+        //anchors.horizontalCenter: parent.horizontalCenter
+        // source: "qrc:/gcompris/src/GianaBhateja/resource/img2.svg";
+        //sourceSize.height: visible ? 80 * ApplicationInfo.ratio : 1
+
+        anchors.fill: parent
+       // color: "#ABCDEF"
+        signal start
+        signal stop
+
+        Component.onCompleted: {
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        GCText {
+            id: text
+            anchors {
+                left: parent.left
+                top: bar.top
+                bottom: parent.bottom
+                margins: 10
+            }
+            text: "More Games are coming soon. Stay tuned... :) "
+            fontSize: smallSize
+        }
+
+
+
+        TextEdit {
+            id: edit
+            textFormat: TextEdit.RichText
+            anchors {
+                right: parent.right
+
+                margins: 10
+            }
+            x: 500
+            y: 500
+            font {
+                pointSize: (18 + ApplicationSettings.baseFontSize) * ApplicationInfo.fontRatio
+                capitalization: ApplicationSettings.fontCapitalization
+                weight: Font.DemiBold
+                family: GCSingletonFontLoader.fontLoader.name
+                letterSpacing: ApplicationSettings.fontLetterSpacing
+                wordSpacing: 10
+            }
+
+            function insertText(text) {
+                edit.insert(parent.BottomLeft, text)
+            }
+        }
+        Column {
+            id: control
+            width: 400 * ApplicationInfo.ratio
+            anchors {
+                left: parent.left
+                top: parent.top
+                bottom: bar.bottom
+                margins: 10
+            }
+            spacing: 10
+
+            Rectangle {
+                id: simpleButton
+                color: "blue"
+                width: 500; height: 75
+
+
+
+                Text {
+                    id: buttonLabel
+                    anchors.centerIn: parent
+                    text: "Drag and Drop rectangles in ascending sequence... :)"
+                }
+
+                MouseArea {
+                    anchors.fill: parent
+                    //drag.target: rect
+                    //drag.axis: Drag.XAndYAxis
+                    onClicked: console.log("Blue coloured rectangle taken")
+                }
+                // Determines the color of the button by using the conditional operator
+                // color: buttonMouseArea.pressed ? Qt.darker(buttonColor, 1.5) : buttonColor
+            }
+            Rectangle {
```

Comment:
You can probably use a Repeater instead of copy/pasting rectangles that have the same behaviour.